ctf/ops/infisical/Dockerfile

```
@@ -0,0 +1,13 @@
+# Infisical — Render private service
+# Self-hosted secrets manager. Provides secret injection for all CTF v3 services.
+# See: https://infisical.com/docs/self-hosting/overview
+#
+# Required env vars (set in Render dashboard):
+#   ENCRYPTION_KEY          — 32-byte hex secret for data-at-rest encryption
+#   AUTH_SECRET             — session signing secret (JWT)
+#   MONGO_URL               — MongoDB Atlas connection string
+#   SITE_URL                — public HTTPS URL of this Infisical service on Render
+#   SMTP_HOST / SMTP_PORT   — optional email (for invite flows)
+FROM infisical/infisical:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)